pkg/skaffold/initializer/init.go

```diff
@@ -654,8 +675,10 @@ func checkConfigFile(filePath string, force bool, potentialConfigs *[]string) (b
 	}
 
 	if IsSupportedKubernetesFileExtension(filePath) {
-		*potentialConfigs = append(*potentialConfigs, filePath)
-		return true, nil
+		if filepath.Base(filePath) != "package.json" { // Not a valid k8s manifest
```

Comment:
This doesn't seem scalable and there must be dozens of JSON and YAML files that caught up in this!

Reply:
Yeah, it was a quick hack. I think the verification of k8s manifests has to be improved